.github/workflows/tests.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           - windows-latest
           - macos-latest
         rust-version:
-          - nightly
+          - stable
```

Comment:
nit: I'd either drop `rust-version` from `matrix`, or better run CI on both `stable` and `nightly` as long as `nightly` CI is ignored and is not blocking.

Reply:
Let's drop it for now!